lib/ssz_ex.ex

```diff
@@ -634,11 +783,16 @@ defmodule LambdaEthereumConsensus.SszEx do
   end
 
   defp next_pow_of_two(len) when is_integer(len) and len >= 0 do
-    if len == 0 do
-      0
-    else
-      n = ((len <<< 1) - 1) |> :math.log2() |> trunc()
-      2 ** n
+    cond do
+      len == 0 ->
+        0
+
+      len == 1 ->
+        1
```

Comment:
These can be moved to the function signature. Also, I think the `len == 1` one isn't needed, since `(1 << 1) - 1 == 1` and `2 ** log(1) == 1`